examples/simple-mcp/weather.py

```
@@ -0,0 +1,111 @@
+from mcp.server.lowlevel import Server
+import anyio
+import mcp.types as types
+import json
+from pydantic import FileUrl, AnyUrl
+from urllib.parse import unquote
+import os
+import sys
+import httpx
+
+BASE_API = "http://t.weather.sojson.com/"
+f = open(os.path.join(os.path.dirname(__file__), "cities.json"), "r")
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)